tests/DotNet.Sdk.Extensions.Testing.Tests/Configuration/AddTestConfigurationWebHostTests.cs

```diff
@@ -3,10 +3,24 @@ namespace DotNet.Sdk.Extensions.Testing.Tests.Configuration;
 [Trait("Category", XUnitCategories.Configuration)]
 public class AddTestConfigurationWebHostTests
 {
+    // On the below summary I'm not using a <see cref=""/> tag for WebHost.CreateDefaultBuilder()
+    // because the static WebHost exists on both Microsoft.AspNetCore and Microsoft.AspNetCore.Hosting
+    // and I have both of then as global usings.
```

Comment:
This can be resolved by placing `using WebHost = [whichever one];` at the top of the current file. The `cref` will resolve against the explicit alias before falling back to global usings where the conflict is.

Reply:
Thanks for the tip 👏 . Totally forgot that I could use an alias to solve this.